core/variant_call.cpp

```diff
@@ -284,8 +284,8 @@ struct _VariantCall {
 	VCALL_LOCALMEM0R(String, c_escape);
 	VCALL_LOCALMEM0R(String, c_unescape);
 	VCALL_LOCALMEM0R(String, json_escape);
-	VCALL_LOCALMEM0R(String, percent_encode);
-	VCALL_LOCALMEM0R(String, percent_decode);
+	VCALL_LOCALMEM0R(String, uri_encode);
```

Comment:
This breaks compatibility with 3.0, so we can't merge it as is. You can add a `uri_encode`, but the old methods should stay (they can be marked as deprecated though and removed later on).

Reply:
If 3.0 and 3.1 have to be compatible, then I think it's better to keep the percent_encode name. A bit less appealing to me, but not bad either.